dips/dip-1.mdx

```diff
@@ -211,18 +211,88 @@ Represents an error that occurred in response to a Command.
 {
     "type": "command_error",
     "field": "payment.sender.kyc_data.surname",
-    "code": "missing_data",
+    "code": "missing_field",
     "message": "",
 }
 ```
-`command_error` occurs in response to a Command failing to be applied - for example, a high level validation error.  `protocol_error` occurs in response to a failure related to the lower-level protocol.  Note: Additional information detailing all errors will be added.
 
+`command_error` occurs in response to a Command failing to be applied - for example, a high level validation error.  `protocol_error` occurs in response to a failure related to the lower-level protocol.
+
+#### List of Error Codes
```

Comment:
how do we support business errors? Cover all with a "rejected" ?

Reply:
If you are talking about set actor.status.status to abort with code reject, then no, because "rejected" is the code the status abort.